tests/test_unit/test_singleton.py

```diff
@@ -0,0 +1,10 @@
+from unittest import TestCase
+from gamebench_api_client.singleton import Singleton
+
+
+class TestSingleton(TestCase):
```

Comment:
```suggestion
class SingletonStub(Singleton):
    def __init__(self):
        self.property_a = None
        self.property_b = None
    
    def set_property_a(self, input)
        self.property_a = input
        
    def set_property_b(self, input)
class TestSingleton(TestCase):
    def test_singleton_pattern(self):
        this_single = SingletonStub()
        this_single.set_property_a("1st")
        
        that_single = SingletonStud()
        that_single.set_property_a("2nd")
        
        self.AssertIs(this_single, that_single)
        
```

Reply:
I changed the test in d9378fd to check that the class is indeed a singleton instead of just testing that it can be created.